roles/cockpit-ui/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Expose registry with route
+  command: oc expose service docker-registry
+
+- name: Install Cockpit template
+  command: oc create -f registry-console.yaml -n default
+
+- name: Create passthrough route for Registry
+  command: oc create route passthrough --service registry-console --port registry-console -n default
+
+- name: Deploy Registry
+  command: oc new-app -n default --template=registry-console -p OPENSHIFT_OAUTH_PROVIDER_URL="{{ openshift_https_proxy }}:8443",REGISTRY_HOST=$(oc get route docker-registry -n default --template='{{ .spec.host }}'),COCKPIT_KUBE_URL=$(oc get route registry-console -n default --template='https://{{ .spec.host }}')
+
+- name: Patch Cockpit routes
```

Comment:
Not necessary. We're using a passthrough route with auto-generated SSL certs. Custom certs are post install per docs PR https://github.com/openshift/openshift-docs/pull/2696